dapps/digital-twin-detail/src/i18n/de.json

```diff
@@ -74,10 +74,22 @@
       "transactions-title": "Letzte Transaktionen"
     },
     "sharings": {
-      "sharings-title": "Freigaben"
+      "filters": {
+        "all": "Alle",
+        "company": "Unternehmen",
+        "unshared": "Unbekannt",
+        "user": "Nutzer"
+      },
+      "no-sharings": "Dieser Digitale Zwilling wurde mit keinem Nutzer geteilt.",
+      "no-sharings-filter": "Dieser Digitale Zwilling wurde mit keinem Nutzer mit diesem Filter gefunden. (Suche: \"{searchTerm}\" und Filter: \"{filter}\")",
```

Comment:
Is the part in parenthesis necessary? Looks kinda convoluted. Better ask the concept people :)